Sources/Extensions/TableDirector/TableDirector+Extensions.swift

```diff
@@ -112,30 +112,3 @@ public extension TableDirector {
     }
 
 }
-
-// MARK: - Separator Configuration
-
-public extension TableDirector {
```

Comment:
file is removed, but it exists in `project.pbxproj` file

Reply:
It does't remove

Reply:
ok. my mistake